style.css

```diff
@@ -78,6 +119,50 @@ p {
   justify-content: space-between;
 }
 
+.container {
+  height: 474px;
+  margin-left: 20px;
+  margin-right: 20px;
+  background-color: #ebf0ee;
+  border: 1px solid #d0d9d4;
+  border-radius: 8px;
+  display: grid;
+  grid-template-rows: 252px 89px;
+}
+
+.container2 {
+  height: 352px;
+  margin-left: 20px;
+  margin-right: 20px;
+  background-color: #ebf0ee;
+  border: 1px solid #d0d9d4;
+  border-radius: 8px;
+  display: grid;
+  grid-template-rows: 184px 32px 136px;
+}
```

Comment:
Kindly make sure that you are using flexbox to organize the elements inside the skills cards. This is part of the requirements of the project. :eyes: :eyes: 
![image](https://user-images.githubusercontent.com/98363075/198797924-a853ffd7-804e-4a8a-80d4-80607cdab472.png)